Templates/TodoList-NodeJS/index.js

```diff
@@ -0,0 +1,45 @@
+﻿module.exports = function (context, req, inputList) {
+    var method = req.method.toLowerCase();
+
+    if (methods[method]) {
+        methods[method](context, req, inputList);
+    } else {
+        context.res = { status: 404, body: "No route found for VERB: " + req.method };
```

Comment:
Once you put your methods restriction in the binding, you won't need this validation check.